core/templates/pages/admin-page/services/admin-router.service.ts

```diff
@@ -25,11 +25,18 @@ import { AdminPageConstants } from
 @Injectable({
   providedIn: 'root'
 })
+
 export class AdminRouterService {
   currentTabHash: string = (
     AdminPageConstants.ADMIN_TAB_URLS.ACTIVITIES);
 
+  /**
+   * @param {string} tabHash tab url hash value.
```

Comment:
For all docstring descriptions: please start with a capital letter.

Also, try to make them clearly readable. If you just say "tab url hash value", I don't think that developers will clearly understand it. Use full English sentences/clauses instead, don't abbreviate things or leave out words. Try to explain it to someone who doesn't know the code.

Reply:
Sure ill change it

Reply:
Thanks -- just a tip, though: please make the changes first, and then submit the comments to these at the same time as you submit the code. 

In general, please make sure to follow the instructions [here](https://github.com/oppia/oppia/wiki/Contributing-code-to-Oppia#instructions-for-making-a-code-change). Otherwise, it makes it harder for reviewers to keep track of these comments.